packages/RadioGroup/src/index.js

```diff
@@ -1,4 +1,4 @@
 import RadioGroup from './RadioGroup';
 import Radio from './Radio';
```

Comment:
Might we consider doing the following instead? ```JS
export { default as Radio } from './Radio';
```

Reply:
@hswolff thoughts?